src/sensor.h

```diff
@@ -246,6 +252,8 @@ namespace librealsense
         std::shared_ptr<raw_sensor_base> _raw_sensor;
         formats_converter _formats_converter;
         std::vector<rs2_option> _cached_processing_blocks_options;
+
+        options_watcher _options_watcher;
```

Comment:
I see you left the default ctor to be called...
But can we somehow leverage the device settings for the watch interval to use?

Reply:
Added `options-update-interval` parameter to the settings